### ISSUE TITLE
Can't get children to work properly

### ISSUE THREAD

**Issue by process91** [ludovicchabant/PieCrust2#28]
Perhaps I'm misunderstanding the documentation, but here's what I have:
/pages/teaching.html
/pages/teaching/child.html

Both have some content.

When I look at what is returned for family.children in the PieCrust debug info at localhost:8080/teaching?!debug, it is an empty array. Shouldn't I see the content from child.html in there? Am I doing it wrong?

**Comment by process91** [comment]
It seems that accessing the children works fine, it just is not printing in the debug info panel when using ?!debug. I verified this against the docs (in this GitHub repo), 
<a href="http://i.imgur.com/uHy94jP.png">
  <img src="http://imgur.com/uHy94jPl.png" />
</a>
Note that this page does have children, and in fact it is looping through and displaying them in a list, but the family.children object within the debug panel shows (empty).

**Comment by process91** [comment]
Some more information - on line 56 of piecrust/data/linker.py, despite family.children having members (in the template), children = self._linker._self_item._linker_info.child_linker is of None type.

**Comment by ludovicchabant** [comment]
Hi, thanks for reporting this!

I looked into it and I don't get the `None` type you mention in that last comment. It looks to me the bug is really just that PieCrust returns some complex data that the debug window can't really render, resulting in a seemingly empty bunch of properties (`children`, `siblings`, etc.).

I fixed it locally, but I have a bunch of other big changes to sort through first so it will hopefully be pushed here sometime next week.
